README.md

```diff
@@ -1 +1,73 @@
+| Description (Optional)                                   | Lambda function description what will be create                                                                           |
+| Logz.io shipping Token (Required)                        | Your Logz.io metrics account token.                                                                                       |
+| Range time for send data to the logs (Default: 1 minute) | Define a time in a minutes to run a lambda function (using cloudBridge event)                                             |
+
+2. On a Second tab `Aws Settings` you will see. Please fill all reqiured fields.
+
+![UI second screen](assets/second-screen.png)
+
+| Field                                 | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
+| ------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aws Access Key (Required)             | Your AWS access key ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+| Aws Secret Key (Required)             | Your AWS secret key ID.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
We need to tell our users what are the minimum permissions required for the iam user, for this integration to work